### ISSUE TITLE
MobileNetV3 QAT TFLite Conversion Issue

### ISSUE THREAD

**Issue by tarushbansal** [tensorflow/model-optimization#1107]
Prior to filing: check that this should be a bug instead of a feature request. Everything supported, including the compatible versions of TensorFlow, is listed in the overview page of each technique. For example, the overview page of quantization-aware training is [here](https://www.tensorflow.org/model_optimization/guide/quantization/training). An issue for anything not supported should be a feature request.

**Describe the bug**

This is a similar issue to #368 but for MobileNetV3Large, where after Quantisation Aware Training, I see a large drop in accuracy in the QAT TFLite model compared to the corresponding QAT Keras Model. Minor Implementation details: I had to refactor the default MobileNetV3Large Keras Code to make it compatible with QAT in the Tensorflow Model Optimisation library by replacing the Add operations in its Hard Sigmoid function with Rescaling and using a Moving Average Output only Quantiser for the Multiply and Rescaling layers in the network. I train the network for more than 6-7 epochs with ~5100 batches in each epoch (each batch consisting of 10 samples) but I see no convergence between the Keras and TFlite models as was seen in #368. I see a number of people in #974 have raised the same issue but this has not been fixed yet. It's likely that is a kernel implementation bug similar to #368 so would be great if a fix could be developed for this. It might be helpful to note that I didn't face this issue in MobileNetV3Large minimalistic which makes me wonder that the issue might be in the Multiply Layers of the Squeeze-Excite and Hard Swish functions.

Would appreciate any help. Thanks!

**System information**

TensorFlow version (installed from source or binary): 2.15.0

TensorFlow Model Optimization version (installed from source or binary): 0.7.5

Python version: 3.10.12

Keras Version: 2.15.0

**Describe the expected behavior**
QAT Keras Model should generate identical outputs to the converted QAT TFlite Model

**Describe the current behavior**
Converted QAT TFlite Model has much lower accuracy than the corresponding QAT Keras Model

**Code to reproduce the issue**
Provide a reproducible code that is the bare minimum necessary to generate the
problem.

**Screenshots**
![Screenshot 2024-01-15 at 09 57 11](https://github.com/tensorflow/model-optimization/assets/73364892/34c8db85-633a-4175-b4c8-364526f8d375)
![Screenshot 2024-01-15 at 09 57 47](https://github.com/tensorflow/model-optimization/assets/73364892/f0170450-bfb9-45a1-abfa-2a4c27373f1c)

**Additional context**

**Comment by Alexey234432** [comment]
I had the same issue and was not able to solve it; it was specifically for MobileNetV3 with Keras / TFLite. If I remember correctly as a workaround I had to switch to "SLIM" based implementation (w/o Keras at all).

**Comment by tarushbansal** [comment]
Hi @Alexey234432. Just for curiosity, did you have any luck with getting the SLIM based implementation working? It's unfortunate that Keras / TFLite doesn't have a solution for this since MobileNetV3 is quite a popular choice for edge-device use cases these days.

**Comment by doyeonkim0** [comment]
Hi @tucan9389 
Could you please take a look into this? Thank you!

**Comment by MATTYGILO** [comment]
@tarushbansal @Alexey234432 
Has a solution been found for tensorflow keras?

**Comment by MATTYGILO** [comment]
@tucan9389 Has a solution been found?

**Comment by tucan9389** [comment]
@MATTYGILO No.. As far as I know, this thread is up-to-date information.
